### ISSUE TITLE
Latest `mk2utc.clk` differs from version in this repo

### ISSUE THREAD

**Issue by dlakaplan** [ipta/pulsar-clock-corrections#17]
Assume tempo2 repo is more current.

**Comment by dlakaplan** [comment]
```python ./update_clock_corrections.py  --gh-pages ./gh-pages   --file T2runtime/clock/mk2utc.clk 
Meerkat (TEMPO2)     2023-12-04 18:55:35.017 - Validation failed: New version of T2runtime/clock/mk2utc.clk MJDs differ from old version where they overlap in 597 places
```

**Comment by dlakaplan** [comment]
Closed by #12
